tfhe/src/core_crypto/entities/modulus_switched_lwe_ciphertext.rs

```diff
@@ -0,0 +1,143 @@
+use super::lwe_ciphertext::{LweCiphertext, LweCiphertextOwned};
+use crate::core_crypto::commons::ciphertext_modulus::CiphertextModulus;
+use crate::core_crypto::prelude::*;
+
+pub struct PackedModulusSwitchedLweCiphertext {
+    packed_coeffs: Vec<u64>,
+    lwe_dimension: LweDimension,
+    log_modulus: u64,
+    ciphertext_modulus: CiphertextModulus<u64>,
+}
```

Comment:
sorry missed it here, this should be generic over a Scalar type with the UnsignedInteger bound I would think

a docstring would be wecome as well

Reply:
Made generic and documented